Allow Setting Scanner Expiration Time

MichaelM1177 opened the pull request.

MichaelM1177 committed:
Allow setting `scanningSessionDurationInSeconds`

clutter-tech commented:
<!--
  1 failure:  `danger-swift` fa...
  0 warning: 
  
  1 markdown notices
  DangerID: danger-id-Danger;
-->

<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  `danger-swift` failed.
  </td>
    </tr>
  </tbody>
</table>



### Log

<details>


```
/var/folders/00/9j8kyjwn2cdcdnlxlcpqr5gh0000gn/T/danger/39E7D88B-24A8-42CE-8F6C-F5B0E67FFA79/_tmp_dangerfile.swift:98:15: error: incorrect argument labels in call (have 'inline:directory:', expected '_:inline:configFile:strict:quiet:swiftlintPath:markdownAction:')
96 | }
97 | 
98 | SwiftLint.lint(inline: true, directory: "Session")
   |               `- error: incorrect argument labels in call (have 'inline:directory:', expected '_:inline:configFile:strict:quiet:swiftlintPath:markdownAction:')
99 | 

/var/folders/00/9j8kyjwn2cdcdnlxlcpqr5gh0000gn/T/danger/39E7D88B-24A8-42CE-8F6C-F5B0E67FFA79/_tmp_dangerfile.swift:98:41: error: cannot convert value of type 'String' to expected argument type 'SwiftLint.LintStyle'
96 | }
97 | 
98 | SwiftLint.lint(inline: true, directory: "Session")
   |                                         `- error: cannot convert value of type 'String' to expected argument type 'SwiftLint.LintStyle'
99 | 
[31mERROR: Dangerfile eval failed at Dangerfile.swift
[0;0m[31mERROR: Could not get the results JSON file at /var/folders/00/9j8kyjwn2cdcdnlxlcpqr5gh0000gn/T/danger/39E7D88B-24A8-42CE-8F6C-F5B0E67FFA79/danger-response.json
[0;0m
```


</details>

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against bd7a0cc90889fad4426d68a226a24afbc693aef7
</p>

MichaelM1177 committed:
Update for CircleCI maintenance

- Bump ruby
- Bump Xcode
- Delete Cocoapods
- Swap from mint to brew on CircleCI
- Lint via fastlane
